stripe.gemspec

```diff
@@ -33,6 +33,8 @@ Gem::Specification.new do |s|
     /\A\.rubocop/,
     /\A\.travis.yml/,
     /\A\.vscode/,
+    /\Abin/,
```

Comment:
We also ship a script inside a bin directory https://github.com/stripe/stripe-ruby/blob/master/bin/stripe-console, not sure whether it should be removed.

Reply:
Ah I think we should keep it. One sec.